spec/lib/webhooks/stripe/stripe_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+#requires that the webapp is running on port 5000
+
+describe "Stripe Webhooks", :webhooks do
+
+  let(:api_path) { "http://localhost:5000/api/site/stripe/webhook"}
+  let(:stripe_token) { ENV["STRIPE_WEBHOOK_TOKEN"] }
```

Comment:
Stripe doesn't provide any mechanism whatsoever to auth webhooks, so its either that or we stash a key in couch.